src/components/dashboard/SendTokens/Dash/Form.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { I18n } from 'react-redux-i18n'
+import { FormControlLabel } from '@material-ui/core'
+import { Checkbox } from 'redux-form-material-ui'
+import { Field, reduxForm } from 'redux-form/immutable'
+import { estimateFee } from '@chronobank/core/redux/dash/thunks'
+
+import { FORM_SEND_TOKENS } from 'components/constants'
+import BitcoinLikeBlockchainForm, { mapStateToProps } from '../BitcoinLikeBockchain/Form'
+import { prefix } from '../lang'
+import validate from '../validate'
+
+function mapDispatchToProps (dispatch) {
+  return {
+    estimateFee: (params) => dispatch(estimateFee(params)),
+  }
+}
+
+class DashForm extends BitcoinLikeBlockchainForm {
+  renderExtraFields () {
+    return (
+      <FormControlLabel
+        control={<Field component={Checkbox} name='instantSend' color='primary' />}
+        label={I18n.t(`${prefix}.instantSend`)}
+      />
+    )
+  }
+}
+
+const DashReduxForm = reduxForm({ form: FORM_SEND_TOKENS, validate })(DashForm)
```

Comment:
Please, use decorators

Reply:
Why? What are the benefits? BTW decorators cannot be always used (they can be used for classes only, decorators cannot be used e.g. for stateless components)?

Reply:
try like this:
```
@connect(mapStateToProps, mapDispatchToProps)
@reduxForm({ form: FORM_2FA_WALLET })
export default class TwoFaWalletForm extends PureComponent {
 
```